tensorflow-core/tensorflow-core-api/pom.xml

```diff
@@ -0,0 +1,321 @@
+    </dependency>
+    <dependency>
+      <groupId>com.google.guava</groupId>
+      <artifactId>guava</artifactId>
+      <version>28.0-jre</version>
+      <optional>true</optional> <!-- for compilation only -->
+    </dependency>
+    <dependency>
+      <groupId>com.squareup</groupId>
+      <artifactId>javapoet</artifactId>
+      <version>1.11.1</version>
+      <optional>true</optional> <!-- for compilation only -->
+    </dependency>
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <version>4.12</version>
+      <scope>test</scope>
+    </dependency>
+  </dependencies>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>build-helper-maven-plugin</artifactId>
+        <version>3.0.0</version>
+        <executions>
+          <execution>
+            <id>add-source</id>
+            <phase>generate-sources</phase>
+            <goals>
+              <goal>add-source</goal>
+            </goals>
+            <configuration>
+              <sources>
+                <source>${project.basedir}/src/gen/java</source>
+              </sources>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+      <plugin>
+        <artifactId>maven-resources-plugin</artifactId>
+        <version>3.1.0</version>
+        <executions>
+          <execution>
+            <id>javacpp-parser</id>
+            <phase>generate-sources</phase>
+            <goals>
+              <goal>resources</goal>
+            </goals>
+          </execution>
+        </executions>
+      </plugin>
+      <plugin>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <version>3.8.0</version>
+        <configuration>
+          <source>1.7</source>
+          <target>1.7</target>
+        </configuration>
+        <executions>
+          <execution>
+            <id>default-compile</id>
+<!-- Bazel currently runs this
+            <configuration>
+              <annotationProcessors>
+                <annotationProcessor>
+                  org.tensorflow.processor.operator.OperatorProcessor
+                </annotationProcessor>
+              </annotationProcessors>
+            </configuration>
+-->
+          </execution>
+          <execution>
+            <id>javacpp-parser</id>
+            <phase>generate-sources</phase>
+            <goals>
+              <goal>compile</goal>
+            </goals>
+            <configuration>
+              <includes>
+                <include>org/tensorflow/c_api/presets/*.java</include>
+                <include>org/tensorflow/processor/operator/OperatorProcessor.java</include>
+              </includes>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+      <plugin>
+        <groupId>org.bytedeco</groupId>
+        <artifactId>javacpp</artifactId>
+        <version>1.5.2-SNAPSHOT</version>
```

Comment:
Do we all have access to JavaCPP 1.5.2-SNAPSHOT? Can we use a released version of JavaCPP available on Maven central instead?

Reply:
I think I can rebase them on a released version yes. In any case, all snapshots are available on Sonatype OSS as well: https://oss.sonatype.org/content/repositories/snapshots/org/bytedeco/